src/integrations/terminal/TerminalManager.ts

```diff
@@ -153,18 +153,22 @@ export class TerminalManager {
 	}
 
 	runCommand(terminalInfo: TerminalInfo, command: string): TerminalProcessResultPromise {
+		// console.log(`[TerminalManager] Running command on terminal ${terminalInfo.id}: "${command}"`)
```

Comment:
Purposely commented out a lot of `console logs ` in this PR because I would love  to keep these debug statements for future terminal issues.